ADAL/src/request/ADAuthenticationRequest+AcquireToken.m

```diff
@@ -183,11 +191,53 @@ - (BOOL)checkExtraQueryParameters
     return url!=nil;
 }
 
+- (BOOL)checkClaims:(ADAuthenticationError *__autoreleasing *)error
+{
+    if ([NSString adIsStringNilOrBlank:_claims])
+    {
+        return YES;
+    }
+    
```

Comment:
Just double-checking: if claims parameter is not set, is it allowed to pass claims in EQP?